man/data.table.Rd

```diff
@@ -110,7 +110,7 @@ data.table(\dots, keep.rownames=FALSE, check.names=FALSE, key=NULL, stringsAsFac
 
     \emph{Advanced:} In the \code{X[Y, j]} form of grouping, the \code{j} expression sees variables in \code{X} first, then \code{Y}. We call this \emph{join inherited scope}. If the variable is not in \code{X} or \code{Y} then the calling frame is searched, its calling frame, and so on in the usual way up to and including the global environment.}
 
-    \item{keyby}{ Same as \code{by}, but with an additional \code{setkey()} run on the \code{by} columns of the result, for convenience. It is common practice to use `keyby=` routinely when you wish the result to be sorted.}
+    \item{keyby}{ Logical scalar indicating if groups defined in \code{by} argument should be sorted. If \code{by} is missing, then \code{keyby} can also take the same input as \code{by} would, but results will be sorted.}
```

Comment:
In addition to allowing `keyby=` take TRUE/FALSE, this PR seems to tilt the emphasis too: the default for `keyby=` in the `[.data.table` signature is now `FALSE` and the argument description here starts with 'Local scalar ...'. Combined, these changes convey, to me at least, that we prefer `keyby=TRUE/FALSE` now.
1) Can the default for `keyby` be restored to be nothing, as it was before. That way, the `keyby=FALSE` doesn't convey that we prefer `TRUE/FALSE`.  I still much prefer seeing `keyby=<expression>`,  rather than both `by=` and `keyby=`.
2) Can this manual page item be changed back to how it was, with an extra sentence at the end saying that `keyby=` can also be TRUE/FALSE too when `by=` is provided.